### PR TITLE
Implementation of details pop up window

### PR DESCRIPTION
**Pull request by shaaibu7**
- Add details pop up window for the card section.
- Serve dynamic content for the project section.

### CODE REVIEW

**Comment by mirouhml** [comment]
Hi @shaaibu7 @Caren-Koroeny, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a complete README, please know that all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present, the only optional section is FAQ, also, make sure to add the section Future Features, including 1-3 potential features to be added, you also need to modify the README and include your information and your partner's.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: app.js [hunk 1]
```diff
@@ -13,4 +16,268 @@ navLinks.forEach((link) => {
...
+      liveImage: 'images/see-live.png',
+      sourceImage: 'images/see-source.png',
+      cancelImage: 'images/xicon.png',
+    },
+    btn1: 'See live',
+    btn2: 'See source',
+    navigation: {
+      live: 'https://github.com/shaaibu7/Portfolio-project',
+      source: 'https://github.com/shaaibu7/Portfolio-project',
+    },
```
Comment (brainconnect93) [inline app.js]: - [ ] Great effort working on your pop-up modal 👏 👏, but I would kindly suggest you have only one `array` of an `object` for creating both your main page dynamically and your pop-up page. with this, you will follow the `DRY (Don't Repeat Yourself)`  method of coding practices. Having only one `array of an object`, you will loop through it over again for the pop-up modal using another value.

File: app.js [hunk 2]
```diff
@@ -13,4 +16,268 @@ navLinks.forEach((link) => {
...
+            <ul class="skills">
+                <li class="html stack"><button>${item.stack[0]}</button></li>
+                <li class="css stack"><button>${item.stack[1]}</button></li>
+                <li class="javascript stack"><button>${item.stack[2]}</button></li>
+            </ul>
+        </div>
+        <div class="click">
+            <button btn-id="${item.id}" class="btn showBtn" type="button">${item.button}</button>
+        </div>
+    </div>
```
Comment (brainconnect93) [inline app.js]: - As requested in the project requirements:


  
- [ ] Kindly remember that you need to follow the original design 😉. Currently, all of your work cards are being displayed in the same way (image on the left & text on the right). Please notice that, in the original design, the project cards alternate between an image on the left & image on the right:

<table>
<th>Your design 👇</th>
</table>

sul-mob

<table>
<th>Figma design 👇</th>
</table>

card

File: style.css [hunk 3]
```diff
@@ -574,6 +575,133 @@ textarea {
...
+/* styling for popup window */
+
+.mainPopup {
+  background-color: #fff;
+  width: 90%;
+  margin: 0 auto;
+  height: 700px;
+  margin-top: 50px;
+  border-radius: 10px;
+}
```
Comment (brainconnect93) [inline style.css]: - [ ] I would kindly suggest you stick to the Figma design as much as possible. Currently, your pop-up cards are not well positioned both in the `mobile` and `desktop` versions respectively. Kindly compare what you have in the Figma template with what you have in your design 👍 👍 

### Highlight
  - Kindly give your desktop pop-up modal card some margin-bottom away from the wall
  - Also I noticed the content on the main page is still showing while the pop-up modal card is displayed which is not meant to be like that. kindly give the pop-up card `container` some `CSS property` to that effect
  e.g ```backdrop-filter: blur(8px);```
  - The mobile version of the pop-up modal card is missing some of the content due to wrong `alignment` and positioning. could you please fix that too

 - Below is a video recording where I point out some errors that need your attention 👇 
 
[screen-capture.webm](https://user-images.githubusercontent.com/78634841/209331970-67e7e84a-ce15-4481-86cc-0dbf9f99a52c.webm)

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
# STATUS: Changes Required :recycle:

### Hi @shaaibu7, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @shaaibu7, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well-implemented suggestions.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] You can still adjust your pop-up card design. Kindly try to match your design with the Figma design.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**